Src/SarifPatternMatcher/SearchSkimmer.cs

```diff
@@ -16,13 +16,13 @@ namespace Microsoft.CodeAnalysis.SarifPatternMatcher
 {
     public class SearchSkimmer : Skimmer<AnalyzeContext>
     {
-        private const string Base64DecodingFormatString = @"\b(?i)[0-9a-z\/+]{{{0}}}==";
```

Comment:
>}}}=="; [](start = 81, length = 7)

Oops. This regex specified padding when it shouldn't have. The padding depends on the length arg injected into this format string. We now compute the padding (later).